app/helpers/jwt.js

```diff
@@ -4,3 +4,5 @@ const { secret, expirationToken } = require('../../config').common.session;
 
 exports.getToken = ({ id, firstName, lastName }) =>
   jwt.sign({ id, firstName, lastName }, secret, { expiresIn: expirationToken });
+
+exports.verifiedToken = token => jwt.verify(token, secret);
```

Comment:
To better control errors here, make this function asynchronous using a callback.